gemfiles/rails42.gemfile

```diff
@@ -35,6 +35,7 @@ group :test do
   gem "shoulda-matchers", "~> 2.8.0", require: false
   gem "timecop"
   gem "webmock"
+  gem "pundit"
```

Comment:
You can manually remove these, not too sure why they were added in the first place